diff --git a/node_modules/local-cors-proxy/lib/index.js b/node_modules/local-cors-proxy/lib/index.js
index 6dcbec6..67d7cd2 100644
--- a/node_modules/local-cors-proxy/lib/index.js
+++ b/node_modules/local-cors-proxy/lib/index.js
@@ -20,6 +20,8 @@ var startProxy = function(port, proxyUrl, proxyPartial, credentials, origin) {
     req.pipe(
       request(cleanProxyUrl + req.url)
       .on('response', response => {
+        // Allow client access to all non-standard headers
+        response.headers['access-control-expose-headers'] = "*";
         // In order to avoid https://github.com/expressjs/cors/issues/134
         const accessControlAllowOriginHeader = response.headers['access-control-allow-origin']
         if(accessControlAllowOriginHeader && accessControlAllowOriginHeader !== origin ){
